tests/test_shield_privacy_prefs.py

```diff
@@ -12,16 +13,111 @@ def create_ping_rdd(sc, payload):
     ])
 
 
-def create_row(overrides):
+def create_event_row(overrides):
     keys = ["client_id", "branch", "event", "originDomain", "breakage",
             "notes", "study"]
 
-    overrides['study'] = '@shield-study-privacy'
     overrides['client_id'] = 'aa'
 
     return {key: overrides.get(key, None) for key in keys}
 
 
+def create_state_row(overrides):
+    keys = ["client_id", "branch", "study", "study_state"]
+
+    overrides['client_id'] = 'aa'
+
+    return {key: overrides.get(key, None) for key in keys}
+
+
+def test_study_running(row_to_dict, spark_context):
```

Comment:
There doesn't seem to be a code path that checks whether the `study_state` is within a given whitelist of options. These tests seem pretty redundant.